io.github.elograf.Elograf.yaml

```diff
@@ -0,0 +1,58 @@
+id: io.github.elograf.Elograf
+command: elograf
+rename-desktop-file: elograf.desktop
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 5.15-21.08
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    - BASEAPP_REMOVE_WEBENGINE=1
+modules:
+  - name: xdotool
+    buildsystem: simple
+    build-commands:
+      - make
+      - make install PREFIX=/app 
+    sources:
+    - type: archive
+      url: https://github.com/jordansissel/xdotool/releases/download/v3.20211022.1/xdotool-3.20211022.1.tar.gz
+      sha256: 96f0facfde6d78eacad35b91b0f46fecd0b35e474c03e00e30da3fdd345f9ada
+  - name: PyQtApp
+  - name: nerd-dictation
+    buildsystem: simple
+    build-commands: 
+      - "cp nerd-dictation /app/bin/"
+    sources:
+        - type: git
+          url: https://github.com/ideasman42/nerd-dictation
+  - name: python3-vosk
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+        --prefix=${FLATPAK_DEST} "vosk" --no-build-isolation
+    sources:
+    - type: file
+      url: https://files.pythonhosted.org/packages/62/d5/5f610ebe421e85889f2e55e33b7f9a6795bd982198517d912eb1c76e1a53/pycparser-2.21-py2.py3-none-any.whl
+      sha256: 8ee45429555515e1f6b185e78100aea234072576aa43ab53aefcae078162fca9
+    - type: file
+      url: https://files.pythonhosted.org/packages/00/9e/92de7e1217ccc3d5f352ba21e52398372525765b2e0c4530e6eb2ba9282a/cffi-1.15.0.tar.gz
+      sha256: 920f0d66a896c2d99f0adbb391f990a84091179542c205fa53ce5787aff87954
+    - type: file
+      url: https://files.pythonhosted.org/packages/d4/e2/25a02e009538ed03371db59fca54a278192562fea523e851b4842b4a182f/vosk-0.3.37-py3-none-manylinux_2_12_x86_64.manylinux2010_x86_64.whl
```

Comment:
this is x86_64 only. You should get the tar.gz instead to have it built from source.

Reply:
I don't intend to use the compilation way for this part. It would need to include kaldi and the libs needed for it build, like openfst, clapack... at least.
What I expect is to use to part provided by pypi.org. But I don't know if this is possible or allowed.

Reply:
so how does this work on aarch64 that we build for?

(hint it DOES NOT)